xline-client/src/clients/auth.rs

```diff
@@ -0,0 +1,44 @@
+// TODO: Remove these when the placeholder is implemented.
+#![allow(dead_code)]
+
+use std::sync::Arc;
+
+use curp::client::Client as CurpClient;
+use tonic::transport::Channel;
+use xline::server::Command;
+
+use crate::AuthService;
```

Comment:
It's a client crate, why do we have a service here?

Reply:
The `AuthService` naming came from `tower::Service` trait, which is a abstraction from a request to a respond https://docs.rs/tower/latest/tower/trait.Service.html. Here the `AuthService` is used for token authentication.